scene/3d/mesh_instance_3d.h

```diff
@@ -99,6 +99,8 @@ class MeshInstance3D : public GeometryInstance3D {
 
 	virtual AABB get_aabb() const override;
 
+	void bake_mesh_from_current_blend_shape_mix(Ref<ArrayMesh> p_bake_mesh);
```

Comment:
Overall it looks good, but sounds to me the API would work better if it just creates a new mesh and returns it. As it clears the mesh, it does not make much sense to pass an existing one.

Reply:
What about an optional argument (default to nullptr). If null, it allocates a new ArrayMesh. And add a return value so either way it always returns the baked mesh.

Reply:
Added the mesh return as well as the optional existing mesh para.